components/match2/wikis/tft/match_summary.lua

```diff
@@ -0,0 +1,323 @@
+---
+-- @Liquipedia
+-- wiki=tft
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Abbreviation = require('Module:Abbreviation')
+local Class = require('Module:Class')
+local DateExt = require('Module:Date/Ext')
+local Icon = require('Module:Icon')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Page = require('Module:Page')
+local Table = require('Module:Table')
+local String = require('Module:StringUtils')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
+local MatchSummary = Lua.import('Module:MatchSummary/Base')
+
+local MAP_VETO_START = '<b>Start Map Veto</b>'
+local ARROW_LEFT = '[[File:Arrow sans left.svg|15x15px|link=|Left team starts]]'
+local ARROW_RIGHT = '[[File:Arrow sans right.svg|15x15px|link=|Right team starts]]'
+local NONE = '-'
+local TBD = Abbreviation.make('TBD', 'To Be Determined') --[[@as string]]
+
+---@enum WoTMatchIcons
+local Icons = {
+	CHECK = Icon.makeIcon{iconName = 'winner', color = 'forest-green-text', size = '110%'},
+	EMPTY = '[[File:NoCheck.png|link=]]',
+}
```

Comment:
Why not just have these as constants instead of an enum
```suggestion
	local GREEN_CHECK = Icon.makeIcon{iconName = 'winner', color = 'forest-green-text', size = '110%'},
	LOCAL NO_CHECK = '[[File:NoCheck.png|link=]]',
```